src/components/shoppingListItemEditForm/shoppingListItemEditForm.module.css

```diff
@@ -0,0 +1,118 @@
+.root {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  padding: 24px;
+  margin: 0 auto;
+  width: 80%;
+  background-color: #f0f0f0;
+}
+
+.header {
+  font-size: 1.25rem;
```

Comment:
The line height here and in the `.subheader` component looks huge when the description or list title are long.